_includes/manuals/2.1/3.2.2_installer_options.md

```diff
@@ -617,7 +617,7 @@ kafo-export-params -c /etc/foreman-installer/scenarios.d/foreman.yaml -f html
       </tr>
       <tr>
         <td style='white-space:nowrap'>--foreman-ssl</td>
-        <td>Enable and set require_ssl in Foreman settings (note: requires passenger, SSL does not apply to kickstarts)</td>
+        <td>Enable and set require_ssl in Foreman settings (note: requires apache, SSL does not apply to kickstarts)</td>
```

Comment:
I it's typically Apache, but it hasn't been updated yet in the actual parameter.
https://github.com/theforeman/puppet-foreman/blob/495727bdfe4a60df28c542cf5b771a91e20433b3/manifests/init.pp#L64